python/setup.py

```diff
@@ -69,7 +67,8 @@
     ext_modules=cythonize(extensions, language_level="3"),
     zip_safe=False,
     install_requires=["pyarrow>=9,<10"],
-    extras_require={"test": ["pytest>=6.0", "pandas", "duckdb", "click"]},
+    extras_require={"test": ["pytest>=6.0", "pandas", "duckdb", "click"],
+                    "pytorch": ["torch"]},
```

Comment:
I'm wondering if we should just ask the user to manually pip install torch? I feel like Rikai's extras system is way too complicated and hard to maintain. I'm wondering whether we should keep it simple for now.

Reply:
Hmm ok.